setup.json

```diff
@@ -88,7 +88,7 @@
         "aiida_core[atomic_tools]~=1.2",
         "packaging",
         "qe-tools~=1.1",
-        "xmlschema~=1.2"
+        "xmlschema>=1.2.5"
```

Comment:
The correct syntax would be here `xmlschema~=1.2,>=1.2.5` (this allows also versions 1.3, 1.4, ..., but not go do 2.0 that might be backwards incompatible).

Reply:
modified in commit f2f78da3